apps/vaults/components/details/tabs/VaultDetailsStrategies.tsx

```diff
@@ -62,14 +62,20 @@ function	VaultDetailsStrategy({currentVault, strategy}: {currentVault: TYearnVau
 		{revalidateOnFocus: false}
 	) as SWRResponse;
 
-	const result = yDaemonReportsSchema.safeParse(data);
+	let latestApr;
 
-	if (!result.success) {
+	if (!data) {
```

Comment:
This seems needed, as zod will try and validate even when we don't yet have the data.